_rules/SC4-1-2-role-attribute-has-valid-value.md

```diff
@@ -0,0 +1,123 @@
+---
+name: Role attribute has valid value
+test_type: atomic
+
+description: |
+   This rule checks that each role attribute has at least one valid value
```

Comment:
This doesn't check "has at least one", since you can leave it blank. I suggest just saying "has a valid value"